sql-join/big-spenders.sql

```diff
@@ -0,0 +1,7 @@
+select "firstName",
+       "lastName",
+       "amount"
+	from "customers"
+	join "payments" using ("customerId")
+   where "amount" > 10.99
```

Comment:
Try an `order by` instead, since we don't really know the highest prices without looking directly at the data.